packages/edge-gateway/src/durable-objects/summary-metrics.js

```diff
@@ -165,7 +176,14 @@ export class SummaryMetrics0 {
     this.totalWinnerResponseTime += stats.responseTime
     this.totalWinnerSuccessfulRequests += 1
     this._updateContentLengthMetrics(stats)
-    this._updateResponseTimeHistogram(stats)
+    this.responseTimeHistogram = this._getUpdatedHistogram(
+      stats,
+      this.responseTimeHistogram
+    )
+    this.raceResponseTimeHistogram = this._getUpdatedHistogram(
+      stats,
+      this.raceResponseTimeHistogram
+    )
```

Comment:
I don't understand how these differ? They're initialised with the same object, and use the same stats to get updated...